packages/e2e-playwright/src/prompts.js

```diff
@@ -0,0 +1,35 @@
+import { enforcedDevServerPort } from './shared.js';
+
+export default function () {
+  return [
+    {
+      name: 'port',
+      type: 'text',
+      required: true,
+      default: enforcedDevServerPort,
+      message: 'Set the port from which to serve the app:',
+    },
+    {
+      name: 'githubWorkflow',
+      type: 'text',
+      required: true,
+      default: 'y/N',
+      message: 'Do you want to add a GitHub workflow for your CI?',
+    },
+    {
+      name: 'installBrowsers',
+      type: 'text',
+      required: true,
+      default: 'y/N',
+      message:
+        "Install Playwright browsers (can be done manually via 'npx exec playwright install')?",
+    },
+    {
+      name: 'enableCodeCoverage',
+      type: 'text',
+      required: true,
+      default: 'y/N',
+      message: 'Do you want to enable code coverage?',
+    },
```

Comment:
There is no type `text`. It works because it falls back to the default: `input`
Let's use `confirm` type for y/n questions to improve both UX and DX.

```suggestion
    {
      name: 'port',
      type: 'number',
      required: true,
      default: enforcedDevServerPort,
      message: 'Set the port from which to serve the app:',
    },
    {
      name: 'githubWorkflow',
      type: 'confirm',
      required: true,
      default: false,
      message: 'Do you want to add a GitHub workflow for your CI?',
    },
    // TODO: allow selecting which browsers to install
    {
      name: 'installBrowsers',
      type: 'confirm',
      required: true,
      default: false,
      message:
        "Install Playwright browsers (can be done manually via 'npx exec playwright install')?",
    },
    {
      name: 'enableCodeCoverage',
      type: 'confirm',
      required: true,
      default: false,
      message: 'Do you want to enable code coverage?',
    },
```